docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+For the first LST release, the V1.0, all contributors from the beginning of the project will be co-authors. After the
+mention of 'The Gammapy Collaboration', the Lead Developers will be in first, then the past Lead Developers since the
+beginning of the project, followed by the list of contributors in alphabetical order.
+
+General Gammapy publications
+----------------------------
+
+This product aims to describe the project and/or the library as a whole and is submitted to peer-review journals. As it
+targets a wide community, the following scheme is used.
+
+A/ List of authors
+
+A.1/ By default, 'The Gammapy Collaboration' is mentioned first, then the Lead Developers, then the past Lead Developers since the
+last general Gammapy publication, then the list of contributors of each release since the last Gammapy publication.
+
+A.2/ If the editorial rules of the targeted journal permit it, the scheme used by the Astropy project
+(e.g. `Astropy V2.0 <https://arxiv.org/pdf/1801.02634.pdf>`_) should be used in priority:
+
+- 'The Gammapy Collaboration', and the list of primary paper contributors in order of contribution agreed per consensus with the development team and the Coordination Committee, as '(Primary Paper Contributors)'
+- the members of the Coordination Committee, as '(Gammapy Coordination Committee)'
+- the list of contributors of each release since the last Gammapy publication, ordered by alphabetical order, as '(Gammapy Contributors)'
+
+
+A comment on the author list composition should be added. Extracted from the Astropy project, one can precise:
+
+"The author list has three parts: the authors that made significant contributions to the writing of
+the paper in order of contribution, the members of the Gammapy Project Coordination Committee,
+and contributors to the Gammapy Project in alphabetical order. The position in the author list does not
+correspond to contributions to the Gammapy Project as a whole. A more complete list of contributors
+to the core package can be found in the package repository and at the Gammapy team webpage."
+
+C/ The corresponding author is the 'Gammapy Coordination Committee', associated with its usual mailing list.
+
+D/ Acknowledgement
+
+One has the freedom to precisely mention acknowledgements associated with the publication. 
+In practice, it is recommended to precise the grants or fellowships given to some authors.
+One should in any case always acknowledge the Astropy project, to which we are affiliated.
+
+Contributions in conferences
+-----------------------------
+
+The section concerns any contribution in conferences (a talk, a poster and their associated proceedings) related to
+the Gammapy project itself. It does not concern a technical or scientific work that uses Gammapy as an open library.
+In this case, the standard citation scheme should be used.
+
+As the length of the author list is generally a constrain, the author list can be reduced to the shortlist of authors,
```

Comment:
Any proposals on choosing this short author list, and/or the ordering of the names? 

So proceedings would be published as `Presenting author, lead devs, other authors, et al for the gammapy team` ?

Reply:
or authors (PS: a conference contribution can be a poster), lead devs, et al. for the Gammapy collaboration...
A matter of choice;)

Reply:
I will add examples, good idea.
One should still converge towards an order;)